.github/workflows/pr-vrt.yml

```diff
@@ -1,9 +1,8 @@
-name: VRT CI
+name: VRT PR
 on:
   pull_request:
-    # TODO: once testing is done enable pull_request on all branches again
     branches:
-      - vrt-gha-testing
+      - master
```

Comment:
lets remove this, we have various branches that would need to use this pipeline

Reply:
Note that as of now we are running baseline only on pushes to the master branch. If you want this to run on every PR then you will need to update the baseline logic to run on all branches as well, as the current logic finds baseline from the appropriate base branch of the pull request.

I will update both the baseline and PR pipeline.

Reply:
Actually, not sure how this will work. Running baseline on every push would cause too many runs. I think we should figure out a list of branches where we want to run this and add it here and in baseline pipeline.

Or alternatively if you are ok with always running diff against master we can update the logic to only look for baselines from the master branch. It might add some unexpected diffs though as master branch might diverge from the feature branch against which PR is raised.

Reply:
Updated